wtf.html

```diff
@@ -0,0 +1,6 @@
+<html>
+<header><title>wtf</title></header>
```

Comment:
Nice                  spacing   and   code              style

![bitmoji](https://render.bitstrips.com/v2/cpanel/38959507-c24d-47ef-94f7-1420aaca15c2-bc4deb2f-825d-4fea-a2e2-f1212f9c0250-v1.png?transparent=1&palette=1&width=246)

Reply:
thanks dud!

Reply:
Wow wtf is that comment of mine, that's so harsh